koku/masu/external/downloader/azure/azure_report_downloader.py

```diff
@@ -140,64 +148,77 @@ def _get_manifest(self, date_time):  # noqa: C901
             (Dict): A dict-like object serialized from JSON data.
 
         """
-        report_path = self._get_report_path(date_time)
         manifest = {}
-        try:
-            json_manifest = self._azure_client.get_latest_manifest_for_path(report_path, self.container_name)
-        except AzureCostReportNotFound as ex:
-            json_manifest = None
-            msg = f"No JSON manifest exists. Using latest cost export. {ex}"
-            LOG.debug(msg)
-
-        if json_manifest:
-            report_name = json_manifest.name
-            last_modified = json_manifest.last_modified
-            LOG.info(log_json(self.tracing_id, f"Found JSON manifest {report_name}", self.context))
-
-            # Download the manifest and extract the list of files.
+        if self.ingress_reports:
+            report = self.ingress_reports[0].split(f"{self.container_name}/")[1]
+            year = date_time.strftime("%Y")
+            month = date_time.strftime("%m")
+            dh = DateHelper()
+            billing_period = {
+                "start": f"{year}{month}01",
+                "end": f"{year}{month}{dh.days_in_month(date_time, int(year), int(month))}",
+            }
             try:
-                manifest_tmp = self._azure_client.download_file(
-                    report_name, self.container_name, suffix=AzureBlobExtension.json.value
-                )
-            except AzureReportDownloaderError as err:
-                msg = f"Unable to get report manifest for {self._provider_uuid}. Reason: {str(err)}"
+                blob = self._azure_client.get_blob(report, self.container_name)
+            except AzureCostReportNotFound as ex:
+                msg = f"Unable to find report. Error: {str(ex)}"
```

Comment:
The `__str__` method is called inside format strings by default.
```suggestion
                msg = f"Unable to find report. Error: {ex}"
```